### PR TITLE
Fix capsule install failure when generating capsule certs

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: playbooks/roles/capsule/tasks/install.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@
   register: pulp_oauth_secret
 
 - name: 'Generate Certs'
-  command: capsule-certs-generate --capsule-fqdn {{ ansible_nodename }} --certs-tar {{ capsule_certs_tar }} --disable-system-checks
+  command: capsule-certs-generate --capsule-fqdn {{ ansible_nodename }} --certs-tar {{ capsule_certs_tar }}
```
Comment (stbenjam) [inline playbooks/roles/capsule/tasks/install.yml]: This really fixes it? Doesn't capsule-cert-generate still use our hooks directory?
https://github.com/Katello/katello-installer/blob/master/bin/capsule-certs-generate#L52

Comment (ehelms) [inline playbooks/roles/capsule/tasks/install.yml]: Maybe I saw this with the broken capsule-certs-generate where INSTALLER_DIR was not being properly set by the installer. This should still be merged as the command shouldn't require this to be set, and instead we should update the script/installer to handle this properly.

Comment (ehelms) [inline playbooks/roles/capsule/tasks/install.yml]: Proper fix -- https://github.com/Katello/katello-installer/pull/401

**Comment by stbenjam** [comment]
ACK

**Pull request merged**